client/state/posts/reducer.js

```diff
@@ -261,7 +263,16 @@ export function edits( state = {}, action ) {
 				}
 			} );
 
-		case POST_EDITS_RESET:
+		case EDITOR_START:
+			return Object.assign( {}, state, {
+				[ action.siteId ]: {
+					...state[ action.siteId ],
+					[ action.postId || '' ]: { type: action.postType }
+				}
+			} );
+
+		case EDITOR_STOP:
+		case POST_SAVE_SUCCESS:
```

Comment:
I wondered if clearing edits on success would have a negative impact where we currently depend on `type` to be always present, but realized that this should gracefully fall back to the saved post's `type` property. Nice 👍